### PR TITLE
Added scritps for 8-24 updated-terms (privacy) email

### PR DESCRIPTION
**Pull request by aoby**
cc @jeremydstone

### CODE REVIEW

File: bin/mail/2016-08-24-updated-terms/send-real-mails [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+# 367743 total teachers
+
+../send-to-mailing-list --limit 368000 8-22-updated-terms ./teachers.csv
```
Comment (ashercodeorg) [inline bin/mail/2016-08-24-updated-terms/send-real-mails]: Please increase the limit, this is sufficiently important and teacher signups are accelerating.

Comment (ashercodeorg) [inline bin/mail/2016-08-24-updated-terms/send-real-mails]: Can the file(name) be renamed to `2016-08-22-updated-terms`?

EDIT: Actually, to `2016-08-24-updated-terms`?

**Comment by ashercodeorg** [comment]
LGTM.

File: bin/mail/2016-08-24-updated-terms/send-real-mails (hunk 1 above)
Comment (aoby) [reply bin/mail/2016-08-24-updated-terms/send-real-mails]: Poorva is renaming the template to `2016-08-24-updated-terms`. I'll update this script.

**Pull request merged**